src/app/builder/api/ansyn-api.service.ts

```diff
@@ -80,6 +80,10 @@ export class AnsynApi {
 		this.store.dispatch(new DisplayOverlayAction({ overlay, mapId: this.activeMapId, forceFirstDisplay: true }));
 	}
 
+	setOverlays(overlays: IOverlay[]) {
```

Comment:
Update https://github.com/AnSyn/ansyn-angular1/wiki/Ansyn-public-API#setoverlaysoverlays-overlay wiki
add NOTE : that if user wants to use this method he should call "disableInitialSearch" at constructor

Reply:
Added "I" to interfaces names